source/overview/mattermost-cloud-overview.rst

```diff
@@ -2,81 +2,64 @@
 
-The prorated subscription cost for this user is determined by multiplying the monthly amount ($10 USD) by the number of days they are actively registered in the billing period (20) and dividing by the number of days in the month (30).
+This intentional isolation and dedicated deployment allow Mattermost Cloud Enterprise to offer other privacy-centric features, such as VPC Peering, bring-your-own-domain and certificates, mutual TLS, access to monitoring tools, databases to run custom analysis, and more.
 
-The prorated subscription cost for the user added on day 11 is $6.67 USD.
+Future releases will include an enterprise network integration with VPC Peering, TLS mutual authentication, and additional data residency options.
 
-Similarly, if you make a registered user inactive in the middle of a billing cycle, we will calculate the prorated amount using the formula above to determine the prorated charge.
+To get started with Mattermost Cloud Enterprise, visit: https://mattermost.com/cloud to speak to a Mattermost Account Executive.
 
-Viewing and Managing Billing Details
-------------------------------------
+FAQ
+---
 
-You can view your billing details in **System Console > Billing & Account**.
+How much does Mattermost Cloud Cost?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Subscription
-^^^^^^^^^^^^
+Mattermost Cloud Professional is free for up to 10 users. After ten users, it is $10.00/user/month as a pay-as-you-go subscription. Please contact us for over 1000 users.
 
-The subscription screen outlines the details of your Mattermost Cloud plan, including the cost per user per month and the number of users currently registered in your workspace. You can also find a summary of your last invoice in the same page.
+Mattermost Cloud Enterprise is an annual subscription. Please contact us for a quote.    
 
-Billing History
-^^^^^^^^^^^^^^^
+How can customers trial Mattermost Cloud?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The Billing History screen contains a list of all your monthly invoices and payments. Invoices appear immediately after the billing period ends.
+We offer ten users free as a trial option in our Mattermost Cloud Professional edition.  Customers under 11 users can use Mattermost Cloud Professional free for as long as they need.  All features are provided in this free offering. 
 
-Billing periods begin at 12 AM UTC on the first day of each calendar month and end at 11:59 PM UTC on the final day of the same calendar month, at which time subscription costs are calculated and a credit card charge is automatically attempted. It may take up to a day for our billing systems to calculate, charge your credit card, and issue your invoice. Once charges are completed, the invoice will show as *Paid*. The invoice is a downloadable PDF file.
+Mattermost Cloud Enterprise offers a discounted proof-of-concept for 60 days. The proof-of-concept program includes a dedicated customer engineer.  
```

Comment:
```suggestion
Mattermost Cloud Enterprise offers a discounted proof-of-concept for 60 days. The proof-of-concept program includes a dedicated customer engineer.
```

Reply:
This sounds very interesting - do we have a link to more info around it?

Reply:
Draft in progress: https://docs.google.com/document/d/17J8fgeuG9JRTI26hBhFlxV69ZcV4odLgC6kCixux5NA/edit#